veneer/everest-operator.yaml

```diff
@@ -0,0 +1,33 @@
+defaultChannel: stable-v0
+name: everest-operator
+schema: olm.package
+---
+name: fast-v0
+entries:
+- name: everest-operator.v0.0.0
+- name: everest-operator.v0.9.0
+- name: everest-operator.v0.9.1
+  replaces: everest-operator.v0.0.0
```

Comment:
Shouldn't the dev catalog (main branch) have 0.0.0 as the latest one?

Reply:
We have it this way at the moment. I believe it's correct. In order to install dev catalog, you would provide `--version 0.0.0` during installation